src/ontology/ro.Makefile

```diff
@@ -25,12 +25,42 @@ validate-using-oort: ro-edit.owl
 	touch $@
 
 # ========================================
-# Custom components
+# Custom components and release files
 # ========================================
 
 core.owl: components/core.owl components/bfo-axioms.owl components/bfo-classes-minimal.owl
 	$(ROBOT) merge -i components/core.owl -i components/bfo-axioms.owl -i components/bfo-classes-minimal.owl annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
 
+BFO_TERMS_IN_BASE = BFO_0000050 BFO_0000051 BFO_0000060 BFO_0000062 BFO_0000063 BFO_0000066 BFO_0000067 BFO_0000054 BFO_0000054
+
+# base: A version of the ontology that does not include any externally imported axioms.
+# It is customised here because RO has to _inject_ certain axioms that formally belong to BFO.
+# Counts below are approximate, reflecting the state of RO on 4th June 2025.
+# bfo-axioms.owl: ~7 disjointness axioms that are not in BFO
+# el-constraints.owl: ~2 disjointness axioms that are not in BFO
+# temporal-intervals.owl: ~45 RBOX axioms that are not in BFO (many of which have BFO IDs)
+# bfo-classes-minimal.owl: ~12 subclass axioms from BFO needed for RO to function (NOT INCLUDED IN BASE)
+# generated-axioms.owl: ~9 universal restrictions involving RO
+$(ONT)-base.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC) $(IMPORT_FILES)
+	$(ROBOT_RELEASE_IMPORT_MODE) \
+	reason --reasoner ELK --equivalent-classes-allowed asserted-only --exclude-tautologies structural --annotate-inferred-axioms False \
+	relax \
+	reduce -r ELK \
+	remove \
+		--base-iri http://purl.obolibrary.org/obo/RO_ \
+		$(foreach t, $(BFO_TERMS_IN_BASE), --base-iri http://purl.obolibrary.org/obo/$t) \
+	 	--axioms external --preserve-structure false --trim false \
+	merge \
+		-i $(COMPONENTSDIR)/el-constraints.owl \
+		-i $(COMPONENTSDIR)/temporal-intervals.owl \
+		-i $(COMPONENTSDIR)/bfo-axioms.owl \
+		-i $(COMPONENTSDIR)/generated-axioms.owl \
+		-i $(COMPONENTSDIR)/rolification-axioms.owl \
+	$(SHARED_ROBOT_COMMANDS) \
+	annotate --link-annotation http://purl.org/dc/elements/1.1/type http://purl.obolibrary.org/obo/IAO_8000001 \
+		--ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
+		--output $@.tmp.owl && mv $@.tmp.owl $@
```

Comment:
Nitpick: I will never be convinced that this pattern (`-o $@.tmp.owl && mv $@.tmp.owl $@`) is actually useful (frankly it’s cargo-culting at this point: we perpetuate the pattern only because we see it so much that we convinced ourselves that it must be doing something good), but you do you.

Reply:
Agreed, made that change.